### ISSUE TITLE
[BUG] Packs FAULT status only updated with BLEED page open on ECAM

### ISSUE THREAD

**Issue by paul92ilm** [flybywiresim/a32nx#1593]
**Mod Version**

Master

**Describe the bug**

The FAULT status for both Packs on Overhead Panel is only updated when BLEED page is open on ECAM

**To Reproduce**

1. Start Cold & Dark
2. Put Both Batteries on (ext. power too or not)
3. Look at the Pack 1 and Pack 2 Korrys (Overhead) --> No FAULT display
4. Open BLEED page on ECAM
5. Look at the Pack 1 and 2 korrys (Overhead) --> FAULT display

Same thing when you put APU BLEED on, both Packs stay FAULT until you open the BLEED page

**Expected behavior**

No FAULT on Pack 1 & 2 when APU BLEED on
![image](https://user-images.githubusercontent.com/9373163/97512032-c1a51400-1988-11eb-82ee-c59ebbe55010.png)
FAULT appear when you open BLEED page on ECAM

**Actual behavior**

Pack 1 & 2 FAULT with APU BLEED on
![image](https://user-images.githubusercontent.com/9373163/97511985-a76b3600-1988-11eb-87c2-8cc145ee06cf.png)

**References**

**Additional context**

Was this working before/when did the issue start occurring?

Is this a problem in the vanilla unmodded game?

Discord username (if different from GitHub): Lussion

**Comment by James3221** [comment]
all the pack logic runs when the bleed page is open, it's not so much a bug as much it is the current implementation of the systems. I'm pretty sure there's people working on a full cond-pack overhaul and it should work after that.

**Comment by paul92ilm** [comment]
![image](https://user-images.githubusercontent.com/9373163/97554568-3ef77580-19d7-11eb-877e-3e2e06d3121d.png)
Waiting in that case

**Comment by nikotina91** [comment]
yeah the bleed logic is currently tied to the bleed page and only works when you are looking at it, the wasm system will be in Soon™ and will fix that problem (same with a few other pages if you noticed the apu page doesn't update until you look at it).
